node/adapter/netns.py

```diff
@@ -209,3 +218,36 @@ def set_up_endpoint(ip, cpid, next_hop_ips,
         ep.ipv6_nets.add(network)
         ep.ipv6_gateway = next_hop
     return ep
+
+
+def reinstate_endpoint(cpid, old_endpoint, next_hop_ips,
+                       proc_alias=PROC_ALIAS):
+    """
+    Re-instate and endpoint that has been removed.
+    :param cpid: The PID and name of the namespace to operate in.
+    :param old_endpoint: The old endpoint that is being re-instated.
+    :param next_hop_ips: Dict of {version: IPAddress} for the next hops of the
+    default routes namespace.
+    :param proc_alias: The head of the /proc filesystem on the host.
```

Comment:
I'm not sure I understand this comment. Something more on the lines of "the location of the /proc filesystem"